changelog/v1.27.5-patch2/base64url.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+- type: FIX
+  issueLink: https://github.com/solo-io/envoy-gloo/issues/329
+  resolvesIssue: false
+  description: >-
+    Fall back to attempting to decode Base64Url if Base64 decoding fails in the Inja transformation
```

Comment:
```suggestion
    Expose new Base64Url aware transformations that will fall back to attempting to decode Base64Url if Base64 decoding fails in the Inja transformation. These are new options in case users are relying on current behavior and the strict spec for base64
```